src/index.js

```diff
@@ -1,8 +1,10 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import React from "react";
+import ReactDOM from "react-dom";
+import App from "./App";
+import * as serviceWorker from "./serviceWorker";
+import axios from "axios";
 
-ReactDOM.render(<App />, document.getElementById('root'));
+axios.default.baseURL = "http://localhost:3002/api/v1";
```

Comment:
are you running the rails app on localhost:3002?
why?

Reply:
changed it to heroku path